core/src/main/java/org/everit/json/schema/EnumSchema.java

```diff
@@ -111,4 +95,19 @@ protected boolean canEqual(Object other) {
         return other instanceof EnumSchema;
     }
 
+    protected static Object toJavaValue(Object orig) {
```

Comment:
Static methods precede fields, constructors and instance methods everywhere in the codebase. I also can't see the reason for making this method `protected`.